src/features/chat-message/dtos/request/find-chat-messages.request-query-dto.ts

```diff
@@ -0,0 +1,31 @@
+import { ChatMessageModel } from '@src/features/chat-message/mappers/chat-message.mapper';
+import { CursorPaginationRequestQueryDto } from '@src/libs/api/dtos/request/cursor-pagination.request-query-dto';
+import { ParseQueryByColonAndTransformToObject } from '@src/libs/api/transformers/parse-query-by-colon-and-transform-to-object.transformer';
+import { SortOrder } from '@src/libs/api/types/api.constant';
+import { CursorBy, OrderBy } from '@src/libs/api/types/api.type';
+import { IsOptional } from 'class-validator';
+
+type ChatMessagesModelForPaginated = Pick<
+  ChatMessageModel,
+  'id' | 'createdAt' | 'updatedAt'
+>;
+
+export class FindChatMessagesRequestQueryDto extends CursorPaginationRequestQueryDto<ChatMessagesModelForPaginated> {
+  @ParseQueryByColonAndTransformToObject({
+    createdAt: {
+      type: 'date',
+    },
+  })
+  @IsOptional()
+  cursor?: CursorBy<ChatMessagesModelForPaginated, 'id' | 'createdAt'>;
+
+  @ParseQueryByColonAndTransformToObject({
+    createdAt: {
+      enum: SortOrder,
+    },
+  })
```

Comment:
정의된 모델에는 id, createdAt, updatedAt이 있기 때문에

```suggestion
  @ParseQueryByColonAndTransformToObject({
    id: {
      enum: SortOrder,
    },
    createdAt: {
      enum: SortOrder,
    },
    updatedAt: {
      enum: SortOrder,
    },
  })
```
이렇게 해줘야 될 것 같습니다